service/ui/component/Nav.html

```diff
@@ -5,6 +5,7 @@
 		</a>
 		<nav id="proposition-menu">
 			<ul id="proposition-links">
+				{{#if signedInUser}}
```

Comment:
Now only the sign in link works/ is visible, is signedInUser getting passed into this component for it to use?